src/LearningObjectSubmission/interactors/index.ts

```diff
@@ -1,5 +1,6 @@
 import { cancelSubmission } from './cancelSubmission';
 import { checkFirstSubmission } from './checkFirstSubmission';
 import { submitForReview } from './submitForReview';
+import {updateSubmission} from './updateSubmission';
```

Comment:
Little nitpick here for consistency:
```suggestion
import { updateSubmission } from './updateSubmission';
```